src/Misc/Hooks.BugFixes.cpp

```diff
@@ -507,6 +507,16 @@ DEFINE_HOOK(0x73EFD8, UnitClass_Mission_Hunt_DeploysInto, 0x6)
 		return SkipToDeploy;
 	}
 
+	//Force guard mission to mcv when hunt
```

Comment:
I'd rather add a space here
```suggestion
	// Force guard mission to mcv when hunt
```